modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2ActivationKeys.path

```diff
@@ -115,6 +115,49 @@
 	<td>//tbody//*[contains(., '${key_entitlementName}')]/following-sibling::*//*[contains(@class, 'btn')]</td>
 	<td></td>
 </tr>
+
+<!--ACTIVATION_KEY_DETAILS_MODAL -->
+
+<tr>
+	<td>HOST_NAME_VALUE</td>
+	<td>//div[contains(@class, 'row')][5]//*[contains(@class,'col-4')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>IP_ADDRESS_VALUE</td>
+	<td>//div[contains(@class, 'row')][7]//*[contains(@class,'col-4')]</td>
+	<td></td>
+</tr>
```

Comment:
Hi @epark18 is there other approach to retrieve the elements without using the indexes? I ask that because if they change order, the test may fail.